File: Python-MySQL - Without Docker/app/files/models/position_type.py
from files import db


class PositionType(db.Model):
    __tablename__ = 'position_type'
    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
    name = db.Column(db.String(50), nullable=False)

    def __repr__(self):
        return f"{{\"id\": \"{self.id}\", \"name\": \"{self.name}\"}}"

File: Node-MySQL/app/models/PositionType.js
'use strict';
const {
    Model
} = require('sequelize');
module.exports = (sequelize, DataTypes) => {
    class PositionType extends Model {
        /**
         * Helper method for defining associations.
         * This method is not a part of Sequelize lifecycle.
         * The `models/index` file will call this method automatically.
         */
        static associate(models) {
            // define association here
        }
    }
    PositionType.init({
        id: {
            type: DataTypes.INTEGER,
            autoIncrement: true,
            primaryKey: true
        },
        name: {
            type: DataTypes.STRING(50),
            allowNull: false,
            validate: {
                notNull: { msg: "Name required!" }
            }
        }
    }, {
        sequelize,
        modelName: 'PositionType',
        tableName: 'PositionType',
        timestamps: false
    });
    return PositionType;
};